### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by gnandretta**
- Stop ignoring `authParams` during a social login.
- Update font family.

### CODE REVIEW

File: css/index.styl [hunk 1]
```diff
@@ -79,7 +79,7 @@ breakpoint(point, value = 0)
       transform none
 
 .auth0-lock.auth0-lock
-  font-family "Avenir", sans-serif
+  font-family "Avenir Next", Avenir, -apple-system, BlinkMacSystemFont, Roboto, Hevetica, sans-serif
```
Comment (daf-spr) [inline css/index.styl]: I assume that the Font Family declarations for Roboto and the other not-standard fonts is already in place.

File: src/social/actions.js [hunk 2]
```diff
@@ -8,7 +9,7 @@ export function signIn(id, connection) {
 
   const lock = read(getEntity, "lock", id);
 
-  const options = {
+  const options = l.login.authParams(lock).merge(Map({
```
Comment (daf-spr) [inline src/social/actions.js]: Is this use of Map supported? I think it must be used as a constructor

Comment (daf-spr) [inline src/social/actions.js]: Ops... Nevermind, sorry, missed that it comes from immutable

File: css/index.styl (hunk 1 above)
Comment (gnandretta) [inline css/index.styl]: Yes, the Roboto font is available and intended to be used in Android devices.

**Comment by daf-spr** [comment]
:+1: LGTM

**Comment by gnandretta** [comment]
@daf-spr thank you!

**Pull request merged**